lib/flame_audio.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:audioplayers/audioplayers.dart';
+import 'package:audioplayers/audio_cache.dart';
+
+class FlameAudio {
+  AudioCache audioCache = AudioCache(prefix: 'audio/');
+
+  Future<AudioPlayer> play(String file) {
```

Comment:
What about playMusic? to make a pair with playSfx?

Reply:
So your suggestion is to replace `play` by  `playMusic`?

Reply:
Yes, that's my suggestion, what do you think?

Reply:
Agreed!